File: Assignments/2_Operators and Expression/C#/Q3.cs
using System;

class Program
{
    static void Main()
    {
        const double y = 10;
        const double d = 20;

        Console.Write("Enter value of c: ");
        double c = double.Parse(Console.ReadLine());
        Console.Write("Enter value of v: ");
        double v = double.Parse(Console.ReadLine());

        double result = (y - c) / (d + v);
        Console.WriteLine("Result: " + result);
    }
}

File: Assignments/2_Operators and Expression/Javascript/Solutions/Q3.js
function calculateFormula(c, v) {
    const y = 10;
    const d = 20;
    return (y - c) / (d + v);
}

const c = 5;
const v = 3;
const result = calculateFormula(c, v);
console.log(`Result: ${result}`);
